src/LunaDataManager.hx

```diff
@@ -0,0 +1,55 @@
+import rm.managers.DataManager;
+import rm.windows.Window_Message;
+import utils.Fn;
+
+@:keep
+@:native("LunaDataManager")
+class LunaDataManager extends DataManager {
+  public var _testingNewCharacter: String;
+
+  //  #if compileMV
```

Comment:
Is there a reason for all the comments? Or this class?

Reply:
I'm not sure about that one lol I'll remove it 🤣. It was in my thing when I was pulling.

Reply:
All good :D Yea once removed I will accept PR.